public/locales/en/translation.json

```diff
@@ -13,5 +13,26 @@
   },
   "ar": {
     "1": "Arabic"
+  },
+  "whoAreWe": {
+    "header": "who are we?"
+  },
+  "aboutPart": {
+    "0": "We are",
+    "1": " an organization which has a main goal of bringing people happiness, helping them with their mental and other issues. We were found at 2012 by insert_name_here and insert_name_here. Since we are a non profit organization, our expenses are covered by our donators and maybe also you!",
+    "2": " We are here for your empowerment, for you to have a better life.",
+    "3": " We, as SULJAK, are providing a website which is not under the influence of any company. By this we mean, our website does not have any advertisements or any sponsored content."
+  },
+  "nasser": {
```

Comment:
this key should probably be called managerAssistant, the same with jawdat -> generalManager, etc

the reasoning here is that its more likely the names will change than the job titles. for example, in the future, the general manager will change. however, there will still be a general manager. and all you want to do at that point is replace the name.